va_explorer/templates/partials/_top_navigation.html

```diff
@@ -30,6 +30,11 @@
           <a class="{% active '\/(.*)dashboard' %} nav-link" href="{% url 'va_analytics:dashboard' %}">Dashboard</a>
       </li>
       {% endif %}
+      {% if perms.va_analytics.download_data %}
+      <li class="nav-item">
+          <a class="{% active '\/(.*)download' %} nav-link" href="{% url 'va_export:download_form' %}">Download Data</a>
```

Comment:
Thoughts on calling this "Export" instead?

Reply:
Changed to Export